azkaban-webserver/src/main/java/azkaban/webapp/servlet/ScheduleServlet.java

```diff
@@ -684,6 +689,87 @@ private void ajaxScheduleFlow(HttpServletRequest req,
     ret.put("message", projectName + "." + flowName + " scheduled.");
   }
 
+  /**
+   *
+   * This method is in charge of doing cron scheduling.
+   * @throws ServletException
+   */
+  private void ajaxScheduleCronFlow(HttpServletRequest req,
```

Comment:
Thanks for your comments! Yes, I agree that the file is a bit too big. We might refactor it in future.